src/lightkurve/io/eleanorlite.py

```diff
@@ -0,0 +1,114 @@
+"""Reader for GSFC-ELEANOR-LITE light curve files. 
+Details can be found at https://archive.stsci.edu/hlsp/eleanor and https://archive.stsci.edu/hlsp/gsfc-eleanor-lite
+"""
+from ..lightcurve import TessLightCurve
+from ..utils import TessQualityFlags
+from astropy import units as u
+
+from .generic import read_generic_lightcurve
+
+import numpy as np
+
+def read_eleanorlite_lightcurve(filename,
+    time_column="TIME",
```

Comment:
These won't change, we can hard code them, don't need to make them parameters

Reply:
Done